static/templates/user_info_popover_content.handlebars

```diff
@@ -6,6 +6,8 @@
             {{#if is_active }}
                 {{#if is_bot}}
                 <i class="zulip-icon bot" aria-hidden="true"></i>
+                {{else if is_admin}}
+                <img class="crown" src="/static/images/crown.svg" />
                 {{else}}
                 <span class="user-status-indicator popover_user_presence" title="{{user_last_seen_time_status}}"></span>
```

Comment:
Also, it looks like admins will never have a user-presence icon displayed, we should keep both.